packages/ui-extensions/docs/surfaces/point-of-sale/staticPages/pages/pos-overview.doc.ts

```diff
@@ -110,13 +110,18 @@ const data: LandingTemplateSchema = {
           type: 'info',
           sectionContent: `Direct API access is available as of POS version 10.2.0 for extensions targeting \`unstable\`. This feature will be available for all extensions targeting stable API versions in the future.`,
         },
+        {
+          title: 'Access scopes',
+          type: 'note',
+          sectionContent: `Be sure to declare all required access scopes in your app's TOML file. For local development, these access scopes will only be available once the app has been deployed.`,
```

Comment:
I'm not following what you mean by `For local development, these access scopes will only be available once the app has been deployed.` Are you saying if you update the access scopes, the `shopify app dev` command will not reflect this change and you must run `shopify app deploy`?

Reply:
Isn't `deploy` not enough, either? I thought the user would have to visit the admin page for the app in order to get to the dialog that allows approval of new scopes. I'd say let's build this section out to be as helpful as possible.

```suggestion
          sectionContent: `Be sure to declare all required [access scopes](link) in your app's TOML file. Access scopes are only registered or updated when the app is installed, so ensure your app is installed on your test store if your extension relies on Direct Access or Session tokens.`,
```

And then I'd also add a bit more about how to update these scopes.

Reply:
> I'm not following what you mean by `For local development, these access scopes will only be available once the app has been deployed.` Are you saying if you update the access scopes, the `shopify app dev` command will not reflect this change and you must run `shopify app deploy`?

yup exactly. `shopify app dev` does nothing for access scope permissions and requires an app deploy.

> Isn't `deploy` not enough, either? I thought the user would have to visit the admin page for the app in order to get to the dialog that allows approval of new scopes. I'd say let's build this section out to be as helpful as possible.
> 
> And then I'd also add a bit more about how to update these scopes.

cool will add that blurb about installing to test store. will skip how to update scopes since that is too far removed re: direct api. main goal was a note stemmed from the given example.

Reply:
![image](https://github.com/user-attachments/assets/a9abacbc-3b5c-45ff-8a60-41353fdafa4d)